modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/configuration.jsp

```diff
@@ -131,7 +131,7 @@ DLPortletInstanceSettingsHelper dlPortletInstanceSettingsHelper = new DLPortletI
 				<aui:input label="enable-ratings-for-comments" name="preferences--enableCommentRatings--" type="checkbox" value="<%= dlPortletInstanceSettings.isEnableCommentRatings() %>" />
 			</liferay-frontend:fieldset>
 
-			<aui:script require="metal-dom/src/dom as dom">
+			<aui:script require="metal-dom/src/dom as dom, frontend-js-web/liferay/util/build_fragment as buildFragment">
```

Comment:
Sorry about this issue, @boton!

Some comments:
- As @AliciaGarciaGarcia said somewhere (I can't find where, but I have an email to prove it), you don't need `metal-dom/src/dom` here anymore. That should've been removed.
- `buildFragment` is an unnecessary abstraction, you should be able to simply use [Range.createContextualFragment()](https://developer.mozilla.org/en-US/docs/Web/API/Range/createContextualFragment): `document.createRange().createContextualFragment('myString')`. This way you don't have any weird dependency.
- Use `currentColumnsElement.appendChild` instead of `.append` to make the intent more obvious (`append` accepts a string but only to create a TextNode).
- Add `sandbox="<=% true %>"` to the `aui:script` tag once you remove all other dependencies.

Reply:
Don't worry about the issue.

I will do @AliciaGarciaGarcia and your suggestions. thanks to both of you ❤️